cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -315,7 +379,7 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Optional Expand All",
       text =
-        """Optional expand traverses relationships from a given node, and makes sure that predicates are evaluated before producing rows.
+        """The `OptionalExpand(All)` operator traverses relationships from a given node, and ensures that predicates are evaluated before producing rows.
```

Comment:
This could read something like:

OptionalExpand(All) does the same thing as Expand(All) does, except when no relationships match the direction, type and property predicates. In that case, OptionalExpand(all) will return a single row with the relationship and end node are set to `null`

Reply:
👍